File: Pascal/Task90.pas
    program mytime;
    {Задана кількість секунд, що пройшла від початку доби до поточного моменту. 
    Вивести поточний момент часу в годинах, хвилинах та секундах.}
    var t,tm: integer;
    h,m,s:integer;
    begin
        write('Time in seconds: ');
    	readln(t);
    	tm := t div 60;
    	s := t mod 60;
    	h := tm div 60;
    	m := tm mod 60;
    	writeln('Formated time: ', h:2,':',
    		m:2,':', s:2);
    end.

File: Kotlin/Task90.kt
fun main(args: Array<String>) { 
println("Введіть число секунд, що пройшли від початку доби: ") 
// зчитування цілого числа з консолі інструкція вводу даних 
var t: Int = readln().toInt() 
val tm = t / 60 
val s = t % 60 
val h = tm / 60 
val m = tm % 60 
println("%02d".format(h) + ":" + "%02d:%02d".format(m,s)) 
} 